### PR TITLE
Remove travis-ci budge and update links

### PR DESCRIPTION
**Pull request by chraibi**
Can we have a badge from teamcity?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
 # JPScore
 
-[![Build Status](https://travis-ci.org/JuPedSim/jpscore.svg?branch=develop)](https://travis-ci.org/JuPedSim/jpscore)
 [![GitHub license](https://img.shields.io/badge/license-LGPL-blue.svg)](https://raw.githubusercontent.com/JuPedSim/jpscore/master/LICENSE)
-[![DOI](https://zenodo.org/badge/36440436.svg)](https://zenodo.org/badge/latestdoi/36440436)
 
-The core module for performing simulations. See the [online documentation](http://jupedsim.org/jpscore/)
+The core module for performing simulations. See the [online documentation](http://jupedsim.org)
```
Comment (schroedtert) [inline README.md]: . at the end missing

File: README.md [hunk 2]
```diff
@@ -74,7 +72,7 @@ sanitizer enabled. Note there is an approx. 2x slowdown when using
 
 ## Quick start
 
-See [installation and configuration](http://jupedsim.org/jpscore/2016-11-02-quickstart.html)
+See [Getting started with jupedsim](http://www.jupedsim.org/jpscore_introduction.html)
```
Comment (schroedtert) [inline README.md]: . at the end missing

File: README.md (hunk 1 above)
Comment (chraibi) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (chraibi) [reply README.md]: done

**Comment by zeroset** [comment]
> Can we have a badge from teamcity?

Yes we can, but we need to open TC for external network.

**Pull request merged**